### DISCUSSION TITLE
Is it possible to add additional training data during training?

### DISCUSSION THREAD

**Discussion by venominfosec** [coqui-ai/TTS#1285] [General Q&A] [answered]
Is it possible to add additional data during the training process or must all data be present at the start of training and remain unmodified?

Training command if needed: `CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 MKL_NUM_THREADS=1 OMP_NUM_THREADS=1 python3 ~/TTS/TTS/bin/distribute.py --script train_tacotron_dca.py`

**Comment by erogol** [comment, marked as answer]
You need to restart if you add new samples.

**Reply by venominfosec** [reply]
Thank you!
